scripts/README.md

```diff
@@ -6,40 +6,36 @@ A typical release goes like this:
 
 ### Prerequisites
 
-1. You must be a member of the `@mui` org in npm to publish the release
-2. Set up your npm authToken by logging into npm (`npm login`) . This will save a token to `~/.npmrc` as a line that looks
-   like this:
+1. You must be a member of the `@mui` org in npm to publish the release.
```

Comment:
```suggestion
1. You must be a member of the `@base-ui-components` org in npm to publish the release.
```

And we should update once we have @base-ui :)

Reply:
I did miss it, indeed. Will create a follow-up.